### PR TITLE
Upgrade tflint

### PR DESCRIPTION
**Pull request by nvuillam**

### CODE REVIEW

**Comment by echoix** [comment]
I'd prefer another naming convention for easier integration next week with renovate.

Simply the tool name (uppercase), underscore, and VERSION. It is quite standard across projects, and will not put us on our own as we follow a pattern that others use.



Take a look at this commit: https://github.com/echoix/megalinter/commit/fcaa24f96a50dc7b3b2b5b8e266365100b7d7649

And the resulting renovate PR: https://github.com/echoix/megalinter/pull/53/files (it updates the descriptor as well)

**Review by echoix — CHANGES_REQUESTED** [review]
See comment

**Comment by nvuillam** [comment]
@echoix changed :)

File: Dockerfile [hunk 1]
```diff
@@ -44,7 +45,7 @@ FROM checkmarx/kics:alpine as kics
 FROM trufflesecurity/trufflehog:latest as trufflehog
 FROM jdkato/vale:latest as vale
 FROM lycheeverse/lychee:latest-alpine as lychee
-FROM ghcr.io/terraform-linters/tflint:v0.51.0 as tflint
+FROM ghcr.io/terraform-linters/tflint:v${TFLINT_VERSION} as tflint
```
Comment (echoix) [inline Dockerfile:48]: I'm not sure, but maybe I'll have to include the v inside the env var. And anyways, if someone wants to use the variable to point to a different tag, that restricts to tags starting with v

Comment (nvuillam) [reply Dockerfile:48]: I'd like to keep SEMVER values for versions ^^

**Comment by nvuillam** [comment]
@echoix i merged coz i'd like to make a new release today, but we can make updates, like maybe an intermediate variable with docker image name that would be calculated from version semver if not overridden :)

**Pull request merged**